docs/formatting.md

```diff
@@ -25,7 +25,7 @@ multi_formatter_binary(
 
 Assuming you installed with the typical layout:
 
-`bazel run tools:format`
+`bazel run :format`
```

Comment:
Should mention the alias in the `# installation` block above so the targets in the rest of the doc make sense

Reply:
yeah I did that right after you read the file :)